src/components/standalone/monitoring/NetifyInformaticsContent.vue

```diff
@@ -0,0 +1,188 @@
+    )
+    errorLoadingConfiguration.value.notificationDescription = t(getAxiosErrorMessage(exception))
+  }
+}
+
+function clearErrors() {
+  error.value = {
+    notificationTitle: '',
+    notificationDescription: '',
+    uuid: ''
+  }
+}
+
+function handleSaveAutomaticToggle() {
+  clearErrors()
+  saving.value = true
+  const status = form.value.status ? 'enable' : 'disable'
+  ubusCall('ns.netifyd', status)
+    .then((response) => {
+      if (response.data && response.data.result && response.data.result === 'success') {
+        setTimeout(() => {
+          notificationsStore.createNotification({
+            title: t('standalone.netify_informatics.netify_informatics_' + status),
+            description: t(
+              'standalone.netify_informatics.netify_informatics_' + status + '_description'
+            ),
+            kind: 'success'
+          })
+        }, 500)
+      }
+    })
+    .catch((exception: AxiosError) => {
+      errorSaving.value.notificationTitle = t('error.cannot_save_configuration')
+      errorSaving.value.notificationDescription = t(getAxiosErrorMessage(exception))
+    })
+    .finally(() => (saving.value = false))
+}
+
+function open_dashboard() {
+  window.open('https://portal.netify.ai/login', '_blank')
+}
+function copy_uuid() {
+  navigator.clipboard.writeText(form.value.uuid)
+  JustCopied.value = true
+  setTimeout(() => {
+    JustCopied.value = false
+  }, 3000)
+}
+</script>
+
+<template>
+  <div>
+    <FormLayout class="max-w-6xl">
+      <template #description>
+        <i18n-t keypath="standalone.netify_informatics.content_description" tag="p" scope="global">
+        </i18n-t>
+        <NeButton kind="tertiary" size="lg" @click="open_dashboard()" class="ml-2 shrink-0">
+          <template #prefix>
+            <FontAwesomeIcon :icon="['fas', 'arrow-up-right-from-square']" aria-hidden="true" />
+          </template>
+          {{ t('standalone.netify_informatics.open_netify_dashboard') }}</NeButton
+        >
+      </template>
+      <NeSkeleton v-if="loading" :lines="5" />
+      <NeInlineNotification
+        v-if="errorLoadingConfiguration.notificationTitle"
+        class="my-4"
+        kind="error"
+        :title="errorLoadingConfiguration.notificationTitle"
+        :description="errorLoadingConfiguration.notificationDescription"
+      />
+      <div v-if="!isError && !loading" class="mb-8 flex flex-col gap-y-6">
+        <div class="relative flex items-center gap-2">
+          <NeTextInput
+            v-model="form.uuid"
+            disabled
+            :invalid-message="error.uuid"
+            :label="t('standalone.netify_informatics.uuid')"
+            ref="uuidRef"
+          >
+          </NeTextInput>
+          <NeButton kind="tertiary" size="sm" @click="copy_uuid" class="ml-2.5 mt-8">
+            <template #prefix>
+              <font-awesome-icon :icon="['fas', 'copy']" class="h-4 w-4" aria-hidden="true" />
```

Comment:
Same as above, also:
```suggestion
              <FontAwesomeIcon :icon="['fas', 'copy']" class="h-4 w-4" aria-hidden="true" />
```